public/app/features/variables/query/QueryVariableEditor.tsx

```diff
@@ -151,23 +152,29 @@ export class QueryVariableEditorUnConnected extends PureComponent<Props, State>
 
     if (isLegacyQueryEditor(VariableQueryEditor, datasource)) {
       return (
-        <Field label="Query">
+        <Box marginBottom={2}>
+          <div style={{ marginBottom: '8px' }}>
+            <Text element={'h4'}>Query</Text>
+          </div>
```

Comment:
A can this div be replaced with a Box marginTop={1}  that wraps the VariableQueryEditor?

Reply:
Yep makes sense, just pushed the change 👍